clients/src/main/java/org/apache/kafka/common/protocol/types/Struct.java

```diff
@@ -148,6 +148,14 @@ public String getString(String name) {
         return (String) get(name);
     }
 
+    public Boolean getBoolean(Field field) {
```

Comment:
Just thinking it loud here, it seems we follow getter convention here, we really do not have a common convention across the codebase. However, it is not too bad, as this is not end-user. I think user facing classes/interfaces do not follow getter convention though.